sources/styles/app.css

```diff
@@ -0,0 +1,58 @@
+.webixappstart{
```

Comment:
please clean css and leave only the rules you need

Reply:
Done)